README.md

```diff
@@ -5,7 +5,7 @@ Another Kubernetes Helper
 ## Installation
 
 ```sh
-pip install virtualenv
+pip install virtualenv pyyaml
```

Comment:
it's best to not have a diff here at all - docs improvements are actually happening concurrently and the python bits will be addressed 👍